### PR TITLE
6.0.0

### PR DESCRIPTION
**Pull request by sapegin**
Published as 6.0.0-beta.1.

Change log: https://github.com/styleguidist/react-styleguidist/releases/tag/v6.0.0-beta.1

### CODE REVIEW

**Comment by n1313** [comment]
Just to make sure: is first (undocumented) draft of plugins api going live in 6.0, too?

**Comment by sapegin** [comment]
@n1313 No, do you think it’s useful enough? I can merge it.

**Comment by n1313** [comment]
Nah, it's not important. I was just curious which branch I should use if I want to continue tinkering with plugins after 6.0.

File: examples/customised/src/components/Modal/Readme.md [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-        <button onClick={() => setState({isOpen: false})}>Close</button>
-        </Modal>
-    </div>
+```jsx
+<div>
+  <button onClick={() => setState({isOpen: true})}>Open</button>
+  <Modal isOpen={state.isOpen}>
+    <h1>Hallo!</h1>
+    <button onClick={() => setState({isOpen: false})}>Close</button>
+    </Modal>
```
Comment (stepancar) [inline examples/customised/src/components/Modal/Readme.md]: mixed 2 and 4 spaces

Comment (sapegin) [inline examples/customised/src/components/Modal/Readme.md]: Yup, will fix 🍕

**Pull request merged**